test/fixtures/api/swagger.yaml

```diff
@@ -1017,7 +1017,7 @@ definitions:
       - id
       - name
       - url
-      - type
+      - document_category
```

Comment:
As you've seen in  opentrials/api#143, `document_categories` is a table. We can't call that table `types` because: 
* `type` is used a lot throughout the database (for example [in `intervention`](https://github.com/opentrials/api/blob/master/api/models/intervention.js#L14)) where it represents a `string` *attribute* 
* it would mean that we keep all "types" in that table which is not true because we only keep document `type`